packages/devextreme/js/ui/validator.js

```diff
@@ -134,10 +134,18 @@ const Validator = DOMComponent.inherit({
             const isRequired = rules.some(({ type }) => type === 'required') || null;
 
             if(dxStandardEditor.isInitialized()) {
-                dxStandardEditor.setAria('required', isRequired);
+                dxStandardEditor.setAria({
+                    'required': isRequired,
+                    'invalid': isRequired,
```

Comment:
i left a comment below - "With your logic, it is possible to have invalid: true (if data is empty) even if required is false."  the same goes for this line of code. However, if we don't have an initialized value here yet, probably we can omit the invalid attribute. Please check it.